packages/mdc-tabs/README.md

```diff
@@ -284,6 +285,118 @@ import {MDCTabBar, MDCTabBarFoundation} from '@material/tabs';
 const tabBar = new MDCTabBar(document.querySelector('#my-mdc-tab-bar'));
 ```
 
+
+### Using the JavaScript Tab Bar Scroller Component
+
+`mdc-tab-bar-scroller` ships with a Component/Foundation combo for ingesting instances of `mdc-tab-bar`. `mdc-tab-bar-scroller` uses its `initialize()` method call a factory function which gathers and instantiates any tab bar elements that are children of the `mdc-tab-bar-scroller` root element.
```

Comment:
nit: How about "which wraps" instead of "for ingesting", which sounds a bit graphic...